package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dreadnot",
-  "version": "0.2.0",
+  "version": "0.1.4",
```

Comment:
Should probably decide on a version moving forward. There's a version declaration for version 0.1.4 a few lines down but reading the package.json topdown one would thing its v 0.2.0